README.md

```diff
@@ -136,6 +136,17 @@ Options:
                                 Output cannot be used in bash eval.
 ```
 
+## COMPATIBILITY
+
+Bash 4.x and higher is the main target.  You can also use bash 3.2 (mostly for macOS) with the `-G` option
```

Comment:
I used to work on legacy debian version which doesn't have bash 4 nor suitable python version.

Reply:
Also note, that `docopts` work without `-G` nor `-A` and produce global too without prefix. It may be unclear. That was the default behavior in version 0.6.1

Reply:
I can mention legacy Linux - the `mostly` was meant to indicate not just for macOS.

This paragraph is intended to be a brief summary, probably best to have more complete docs elsewhere.